README.md

```diff
@@ -104,6 +105,8 @@ code, other repos are also under active development:
   Heroku in one click.
 - [Kong and Instaclustr](https://www.instaclustr.com/solutions/managed-cassandra-for-kong/): Let
   Instaclustr manage your Cassandra cluster.
+- [Nightly Releases](https://bintray.com/kong/kong-community-edition-nightly/kong-master): Nightly
+  releases are built by travis-ci and available for download.
```

Comment:
Let's also tweak this sentence to mention:
1. A rough time at which the build is available
2. What the build will contain (i.e. the master branch at time of the build)